src/Config_Command.php

```diff
@@ -421,15 +421,50 @@ public function list_( $args, $assoc_args ) {
 	 * @when before_wp_load
 	 */
 	public function get( $args, $assoc_args ) {
-		$path                = $this->get_config_path( $assoc_args );
-		$wp_config_file_name = basename( $path );
-		list( $name )        = $args;
-		$type                = Utils\get_flag_value( $assoc_args, 'type' );
-
-		$value = $this->return_value( $name, $type, self::get_wp_config_vars( $path ), $wp_config_file_name );
+		$value = $this->get_value($assoc_args, $args);
 		WP_CLI::print_value( $value, $assoc_args );
 	}
 
+	/**
+	 * Determines whether value of a specific constant or variable defined is true.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <name>
+	 * : Name of the wp-config.php constant or variable.
+	 *
+	 * [--type=<type>]
+	 * : Type of config value to retrieve. Defaults to 'all'.
+	 * ---
+	 * default: all
+	 * options:
+	 *   - constant
+	 *   - variable
+	 *   - all
+	 * ---
+	 *
+	 * [--config-file=<path>]
+	 * : Specify the file path to the config file to be read. Defaults to the root of the
+	 * WordPress installation and the filename "wp-config.php".
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Get the table_prefix as defined in wp-config.php file.
+	 *     $ wp config get table_prefix
+	 *     wp_
+	 *
+	 * @when before_wp_load
+	 */
+	public function is_true( $args, $assoc_args ) {
+		$value = $this->get_value($assoc_args, $args);
+
+		if ( $value ) {
```

Comment:
I am torn about using `if ( $value )` and `if( boolval($value) )`, as I believe there is more PHP weirdness with `if( $value )`. The primary case I can think of is `if( "0" )` is false, while `if(boolval("0"))` is true.

I think there's a case to be made for either, and believe we should explicitly raise the question when opening the PR, as well as include in test coverage so that it is clear it is _a choice_.

I'm leaning in favor of `if boolval()` as that is more akin to a strict ===, but open to opinions from yourself, @jazzsequence, and Scott not-in-this-repo.

Reply:
```suggestion
		if ( boolval( $value ) ) {
```